dbt/tests/generic/test_res_class_matches_pardat.sql

```diff
@@ -3,17 +3,31 @@
 -- filters for residential parcels by anti-joining the model against `comdat`
 -- using parid and taxyr; as a result, it filters out mixed-use parcels as well.
 --
--- By default, the test will compare the first 3 digits of each set of classes;
--- if `major_class_only=true`, however, the test will compare the first digit
--- only.
+-- Optional parameters:
+--
+-- * major_class_only (bool): Compare only the first digit of classes. When
+-- set to False, compare the first three digits instead.
+-- * parid_column_name (str): The name of the column on the base model that
+-- corresponds to `parid`, in case the model uses a different name scheme.
+-- * taxyr_column_name (str): The name of the column on the base model that
+-- corresponds to `taxyr`, in case the model uses a different name scheme.
+-- * additional_select_columns (dict): Standard parameter for selecting
+-- additional columns from the base model for use in reporting failures.
+-- `model.{column_name}` and `pardat.class` are already selected by
+-- default and do not need to be included using this parameter.
+-- See the `format_additional_select_columns` macro for more information.
+-- Note that while this parameter can often be either a string or a dict,
+-- a dict is required for this test and it must contain the `alias`
+-- and `agg_func` parameters given that results are grouped by `parid`
+-- and `taxyr`.
```

Comment:
**nitpick (non-blocking):** Can we indent these param lists such that all lines except the first start with 2 spaces? (for readability)

Reply:
Annoyingly, removing the indentation is a rule that `sqlfmt` applies automatically. Let me see if I can override it.

Reply:
I went ahead and disabled the formatting for these docstrings in d8bed6c1f7a5de9c020a4f1cd651bf9c147ab489, which is the easiest way to allow for custom formatting given that [sqlfmt doesn't expose any config options other than line length](https://docs.sqlfmt.com/getting-started/configuring-sqlfmt#configuring-style). It works and isn't too invasive; the only downside is that we won't get automatic line-length formatting for docstrings, but I think that's an acceptable price to pay for readability of bulleted lists.